hey_fireball.py

```diff
@@ -19,7 +19,6 @@
 BOT_ID = os.environ.get("BOT_ID")
 EMOJI = os.environ.get('EMOJI')
 POINTS = os.environ.get('POINTS')
-SELF_POINTS = os.environ.get('SELF_POINTS', "DISALLOW")
```

Comment:
Why did you remove getting this env var? It's being used in handle_command.

Reply:
I had some issues with github last night. Should be sorted out now